src/main/java/com/javaschool/mobile/service/AddressServiceImpl.java

```diff
@@ -0,0 +1,12 @@
+package com.javaschool.mobile.service;
+
+import com.javaschool.mobile.dao.AddressDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AddressServiceImpl implements AddressService{
+    @Autowired
```

Comment:
It is better to use contructor dependency than autowired

Reply:
changed DI on constructor